Tests/App/Resources/TestFlows/Throttler/throttler.xml

```diff
@@ -7,15 +7,15 @@
     <camelContext trace="false" xmlns="http://camel.apache.org/schema/spring">
         <route>
             <from uri="test://throttler"/>
-            <throttle timePeriodMillis="60000" asyncDelayed="false">
+            <throttle timeDelaySeconds="1" timePeriodMillis="60000" asyncDelayed="false">
```

Comment:
Is this parameter an apache camel standard?

Reply:
No, but neither is asyncDelayed.